docker/Dockerfile.build.aarch64-qnx

```diff
@@ -30,6 +30,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     pip install clang==14.0 && pip install libclang==14.0.1 && \
     rm -rf /root/.cache/pip/
 
+RUN CMAKE_VERSION=3.20.1 && \
+    wget https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-linux-aarch64.sh && \
+    test -e /bin/sh || ln -s /usr/bin/sh /bin/sh && \
+    chmod +x cmake-${CMAKE_VERSION}-linux-aarch64.sh && \
+    ./cmake-${CMAKE_VERSION}-linux-aarch64.sh --prefix=/usr/local --skip-license && \
+    rm -rf cmake-${CMAKE_VERSION}-linux-aarch64.sh
+
```

Comment:
Is that ever used?

Reply:
Not anymore. I think we can just remove qnx in the next PR.

Reply:
No, I will remove